lib/mutant/matcher/config.rb

```diff
@@ -41,7 +41,7 @@ def inspect
       #
       # @api private
       def add(attribute, value)
-        update(attribute => public_send(attribute).dup << value)
+        with(attribute => public_send(attribute).dup << value)
```

Comment:
I have the same comment for this as https://github.com/mbj/mutant/pull/439/files#r38951709